package.json

```diff
@@ -1,18 +1,31 @@
 {
-  "devDependencies": {
-    "@types/resize-observer-browser": "^0.1.5"
-  },
   "name": "viz-observer",
-  "types": "./types/index.d.ts",
-  "type": "module",
-  "main": "./index.js",
+  "description": "Notifies your code on DOM node move or resize",
+  "author": "Sam Thorogood <sam.thorogood@gmail.com>",
   "version": "1.0.0",
+  "license": "Apache-2.0",
   "repository": "git@github.com:samthor/viz-observer.git",
-  "description": "Notifies your code on DOM node move or resize",
+  "type": "module",
+  "source": "src/index.ts",
+  "module": "./dist/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js"
```

Comment:
I've found putting `types` here too as a conditional export type helps. Just put it in all the places 😂

Reply:
oh didnt know we could do this!